README.md

```diff
@@ -4,28 +4,48 @@
+### Export your design via Framer Generator or via Framer Classic's save function
 
-From the export folder we are only interested in the `imported` subfolder, where you can find the `layers.json` and the `images` folder with all the PNG assets.
+If you never used the Sketch Framer workflow checkout some of guides on Medium like [Sketch Framer Bridge](https://blog.prototypr.io/build-the-bridge-between-sketch-and-framer-a3babf2cfa0f) and [Framer Sketch Workflow](https://medium.com/facebook-design/framer-sketch-an-intentional-workflow-f91ee2ee1cc1), especially on how to best prepare your design in Sketch for the export workflow. Most of it will also be true for later importing to Unity. This primarily regards formatting and naming conventions for easier navigation in all platforms. 
+
+#### Export from Framer Generator (free)
+1. Open Sketch with the relevant file open
+2. Open Framer Generator, and click Sketch > Import
+3. Check the location of the original .sketch file to now see an identically named .framer file next to it.
+4. Proceed to `Layers and Images Extraction from Framer File` section.
+
+#### Export from Framer Classic (trial/paid)
+1. Open Framer Classic, navigate to the Code tab, and click Import in the bottom left (third option up from the bottom)
+2. Select the Sketch file from here that you saved in Sketch earlier - this is the most formal way to import from Sketch to Framer
+3. Now choose File > Save to create the Framer file
+4. Proceed to `Layers and Images Extraction from Framer File` section.
+
+### Layers and Images Extraction from Framer File
+
+1. View that exported framer file's contents (in Mac by right click, Show Package Contents) to show a folder structure that includes the `imported` subfolder.
+2. Navigate into the `imported` subfolder, then into the `[appName@Resolution]` folder, eg. MyAppName@1x.
+3. Copy the `images` folder and `layers.json` file from here into somewhere directly accessible, like the Unity Assets folder.
+4. Proceed to Unity Import section.
 
 ### Import to Unity
 
 * Inside Unity make sure that `Editor Settings/Default Behavior Mode` is set to **2D**, so the assets will be imported as sprites.
 * Open the plugin via `Window/Sketch Import
 * Select the `layers.json` from your *Sketch Export* (inside the `imported` subfolder as mentioned above).
 * Select an UI component (like Canvas,Panel,...) inside your scene as the *Root* for your imported UI layers.
-* Select the Image Prefab from the Import Package Prefabs folder.
+** Note: Selecting Canvas allows you to use "Scale With Screen Size" as the UI scale mode, where the reference resolution can be adjusted so the Sketch file resolution matches the Unity resolution. 
```

Comment:
Formatting is a little off with this additional bullet point and I'm wondering why you removed the point regarding the image prefab. Is this due upgrading to 2018.1 or by accident?

Reply:
Oh, I can change that bullet formatting back. I'm not sure the image prefab needs? I never saw this in the Sketch Import window options. The only visible options were the layers.json selector, my new Scale option, and the root transform. Selecting  the UI component worked fine for that last one, and then the Import button seemed to work correctly from there.

What did the Image Prefab do previously? Any Canvas seems to work as the UI component now - I haven't tested anything else.

Reply:
You are right, this point is outdated. The prefab selection is happening automatically.